File: Assets/AltUnityTester-BindingsAndExamples(can_be_deleted)/Bindings~/java/src/main/java/ro/altom/altunitytester/Commands/ObjectCommand/AltGetText.java
package ro.altom.altunitytester.Commands.ObjectCommand;

import com.google.gson.Gson;
import ro.altom.altunitytester.AltBaseSettings;
import ro.altom.altunitytester.AltUnityObject;
import ro.altom.altunitytester.Commands.AltBaseCommand;

public class AltGetText extends AltBaseCommand {
    private AltUnityObject altUnityObject;
    public AltGetText(AltBaseSettings altBaseSettings, AltUnityObject altUnityObject) {
        super(altBaseSettings);
        this.altUnityObject = altUnityObject;
    }
    public String Execute() {
        String altObject = new Gson().toJson(altUnityObject);
        send(CreateCommand("getText", altObject));
        String data = recvall();
        if (!data.contains("error:")) {
            return data;
        }
        handleErrors(data);
        return "";
    }
}

File: Assets/AltTester/AltDriver/Commands/ObjectCommands/AltGetText.cs
namespace Altom.AltDriver.Commands
{
    public class AltGetText : AltBaseCommand
    {
        readonly AltGetTextParams cmdParams;

        public AltGetText(IDriverCommunication commHandler, AltObject altUnityObject) : base(commHandler)
        {
            cmdParams = new AltGetTextParams(altUnityObject);
        }

        public string Execute()
        {
            CommHandler.Send(cmdParams);
            return CommHandler.Recvall<string>(cmdParams);
        }
    }
}